docs/root/intro/deprecated.rst

```diff
@@ -0,0 +1,6 @@
+Deprecated
```

Comment:
Should we use this? https://github.com/sphinx-contrib/redirects WDYT? This could help with other similar moves in the future?

Reply:
I briefly looked into it, but it wasn't available (or I failed to find it) in pip3, so I gave up.

Reply:
I'm no expert but I think you can just pull directly from GH: https://stackoverflow.com/questions/16584552/how-to-state-in-requirements-txt-a-direct-github-source

Reply:
I played with this for an hour, and I htink the redirects extension is incompatible with the version of sphinx we're using.

I get this error:
```
Traceback (most recent call last):
  File "envoy/build_docs/venv/lib/python3.7/site-packages/sphinx/cmd/build.py", line 283, in build_main
    args.tags, args.verbosity, args.jobs, args.keep_going)
  File "envoy/build_docs/venv/lib/python3.7/site-packages/sphinx/application.py", line 268, in __init__
    self._init_builder()
  File "envoy/build_docs/venv/lib/python3.7/site-packages/sphinx/application.py", line 329, in _init_builder
    self.events.emit('builder-inited')
  File "envoy/build_docs/venv/lib/python3.7/site-packages/sphinx/events.py", line 103, in emit
    results.append(callback(self.app, *args))
  File "envoy/build_docs/venv/src/sphinxcontrib-redirects/sphinxcontrib/redirects/__init__.py", line 30, in generate_redirects
    in_suffix = app.config.source_suffix[0]
KeyError: 0
```

I tried making `source_suffix` into a list in our config, but that didn't fix it. From https://www.sphinx-doc.org/en/master/usage/configuration.html, I think that this field became a hash at some point, so indexing `[0]` won't work with current versions of sphinx.

Reply:
OK np thanks a lot for trying.